excel-importer/src/main/java/uk/gov/hmcts/ccd/definition/store/excel/parser/CaseFieldParser.java

```diff
@@ -50,6 +52,7 @@ public CaseFieldParser(ParseContext parseContext) {
             final CaseFieldEntity caseField = parseCaseField(caseTypeId, caseFieldId, caseFieldDefinition);
             parseContext.registerCaseFieldForCaseType(caseTypeId, caseField);
             caseFields.add(caseField);
+            entityToDefinitionDataItemRegistry.addDefinitionDataItemForEntity(caseField, caseFieldDefinition);
```

Comment:
This is added so that the `CaseField` sheet name is available for label placeholder validation errors.